source/backend/api/Services/DispositionFileService.cs

```diff
@@ -88,6 +94,7 @@ public PimsDispositionFile Update(long id, PimsDispositionFile dispositionFile,
 
             _logger.LogInformation("Updating acquisition file with id {id}", id);
             _user.ThrowIfNotAuthorized(Permissions.DispositionEdit);
+            _user.ThrowInvalidAccessToDispositionFile(_userRepository, _dispositionFileRepository, id);
```

Comment:
Hmm, doesn't this one also need to know if the incoming disposition file is removing the current contractor from the disposition team.

Reply:
that's another story.